src/Form.jsx

```diff
@@ -46,40 +69,45 @@ const Form = ({ formData, setFormData }) => {
         </p>
       </div>
 
-      <IntegerInput
-        label="Stitches per row:"
-        title="The number of stitches in one row"
-        name="crowLength"
-        value={crowLength}
-        setValue={setValue}
-        validator={IntegerInput.validators.nonNegative}
+      <div>
+        <IntegerInput
+          label="Stitches per row:"
+          title="The number of stitches in one row"
+          name="crowLength"
+          value={crowLength}
+          setValue={setValue}
+          validator={IntegerInput.validators.nonNegative}
         />
+      </div>
 
-      <IntegerInput
-        label="Color shift:"
-        title="Start the swatch this many stitches into your color sequence"
-        name="colorShift"
-        value={colorShift}
-        setValue={setValue}
+      <div>
+        <IntegerInput
+          label="Color shift:"
+          title="Start the swatch this many stitches into your color sequence"
+          name="colorShift"
+          value={colorShift}
+          setValue={setValue}
         />
+      </div>
 
-      <IntegerInput
-        label="Number of rows:"
-        title="The number of rows displayed"
-        name="crows"
-        value={crows}
-        setValue={setValue}
-        validator={IntegerInput.validators.nonNegative}
+      <div>
```

Comment:
I’m a bit confused by this div (and also most of the divs wrapping checkbox inputs, but especially this one) These two things don’t seem like they should semantically be together